docs/releases/2.0.txt

```diff
@@ -100,7 +100,12 @@ Minor features
 :mod:`django.contrib.staticfiles`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* Added optional dependency to WhiteNoise which enables Django to serve
```

Comment:
You don't need WhiteNoise if you don't use the staticfiles app. Here it says pytz is not mandatory but highly recommended: https://docs.djangoproject.com/en/1.10/topics/i18n/timezones/ and it is also in `install_requires`. Runserver can be used without the staticfiles app `django.core.management.commands.runserver.Command`. Using the staticfiles app shadows this command by  `django.contrib.staticfiles.management.commands.runserver.Command` and adds the `django.contrib.staticfiles.handlers.StaticFilesHandler` WSGI middleware to catch static files requests. ` The staticfiles runserver command would be deprecated as it is possible to use the original runserver command with WhiteNoiseMiddleware.